### PR TITLE
ENH: Add download link to raw FASTA file

### LINKED ISSUE #35: feature-table view-seq-data should allow for downloading of fasta file
A student at the workshop mentioned that he likes the BLAST links a lot, but would also like to have the fasta file in case he wants to blast some other sequence against his data. This would be easy to add.

### PR DESCRIPTION
**Pull request by jakereps**
Resolves #35

### CODE REVIEW

File: q2_feature_table/_summarize/tabulate_seqs_assets/index.html [hunk 1]
```diff
@@ -8,6 +8,10 @@
         sequence and then click the <i>View report</i>
         button on the resulting page.
       <p>
+      <p>
+        To download a raw FASTA file of your sequences, click
+        <a href="sequences.fasta" download="sequences.fasta">here</a>.
```
Comment (thermokarst) [inline q2_feature_table/_summarize/tabulate_seqs_assets/index.html]: TIL about the [download](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-download) attr!

Comment (jakereps) [inline q2_feature_table/_summarize/tabulate_seqs_assets/index.html]: Same, actually. Do you know if there is a fallback for non-Chrome/Firefox/HTML5 browsers, or is this safe enough for us?

**Comment by jairideout** [comment]
👍 :taco:, there's merge conflicts, ready for merge after that

**Comment by jakereps** [comment]
Thanks @jairideout!

**Pull request merged**